server/controllers/cohort.js

```diff
@@ -353,22 +322,32 @@ const removeMember = (req, resp) => {
         message: 'Member successfully removed from cohort.'
       })
     )
-    .catch(() =>
+    .catch(err => {
+      if (err instanceof BadRequestException) {
+        const { status, message } = err;
+
+        return resp.status(status).send({ message });
+      }
+
       resp.status(400).send({
         message: "Can't remove member from cohort."
-      })
-    );
+      });
+    });
 };
 
-const changeToOwner = (req, resp) => {
+const addOwnerRole = (req, resp) => {
   const { id: cohortId } = req.params;
   const { userId } = req.body;
   const {
     user: { _id: ownerId }
   } = req;
 
   Cohort.findOneAndUpdate(
-    { _id: cohortId, ownerIds: ownerId, memberIds: userId },
+    {
+      _id: cohortId,
+      ownerIds: { $in: [ownerId], $nin: [userId] },
```

Comment:
because here we've got a confusing query.
We're checking if the owner is in the ownerIds array. Logically the owner is always in ownerIds array.

We should check here if currentUser is an owner, and the new user, coming as userId is not an owner - what makes sense while reading.